docs/sidebars.js

```diff
@@ -49,6 +49,7 @@ const sidebars = {
           items: [
             "guides/authentication/custom-auth-upstream",
             "guides/authentication/oauth2",
+            "guides/authentication/cloudentity",
```

Comment:
just comment it out for now

Reply:
Removed in commit <https://github.com/kubeshop/kusk-gateway/pull/796/commits/b7a27035ffdda8721e0c2c5bfbc57e08083a9e65>.